fiftyone/core/multiprocessing/process.py

```diff
@@ -0,0 +1,580 @@
+"""
+Multiprocessing utilities.
```

Comment:
Will remove this for now, just keep the Sequential class and factory and abstract class